packages/block-library/src/query-loop/edit.js

```diff
@@ -97,6 +98,10 @@ export default function QueryLoopEdit( {
 	);
 	const blockProps = useBlockProps();
 
+	if ( ! posts?.length ) {
+		return <div { ...blockProps }> { __( 'No results found.' ) }</div>;
```

Comment:
Why do we need the blockProps in the div? Also could we just turn it into a `p` tag for now?

Reply:
This is the block wrapper. Every block needs at least one element that can act as the block wrapper with all events and attributes attached, otherwise it wouldn't be possible to interact with it and have no toolbar. This block is a light block, so it's responsible for rendering its own wrapper. The block cannot be just a string.

Reply:
But it would be a `p` tag, sure. It can be any tag or component.

Reply:
We need it to mark it as a `block`. If it was just a `string` it wouldn't be selectable and interactive in general. I will make it a `p`.

Reply:
Got it, I didn't review the broader context :)